### PR TITLE
Add options to set CA certificate or use insecure TLS

### LINKED ISSUE #82: Allow setting CA certificate and/or running an insecure https connection
Currently, kube-mgmt can connect to OPA using HTTPS, but it is not possible to do so if OPA's certificate is signed by an internal CA.

In order to support this scenario, kube-mgmt should include flags to either specify a file containing OPA's CA, allow insecure connections, or both.

### PR DESCRIPTION
**Pull request by jutley**
This adds two new flags, `--opa-ca-file` and `--opa-allow-insecure`, in order to enable use cases where OPA uses a certificate signed by an internal certificate authority.

The implementation simple overrides the default TLS configuration, so the rest of the code can continue to rely on the default HTTP client.

Fixes #82

### CODE REVIEW

File: cmd/kube-mgmt/main.go [hunk 1]
```diff
@@ -106,6 +113,27 @@ func run(params *params) {
 		params.opaAuth = strings.Split(string(file), "\n")[0]
 	}
 
+	if params.opaAllowInsecure || params.opaCAFile != "" {
```
Comment (anderseknert) [inline cmd/kube-mgmt/main.go:116]: Would it make sense to make these mutually exclusive? I.e. if a cert is provided, would you still want to allow insecure?

**Review by anderseknert — APPROVED** [review]
LGTM, just a question on the config.

File: cmd/kube-mgmt/main.go (hunk 1 above)
Comment (jutley) [reply cmd/kube-mgmt/main.go:116]: I don't think there is a logical reason to allow both. I'll make them mutually exclusive.

**Comment by jutley** [comment]
Made those options mutually exclusive. Feel free to merge whenever.

**Pull request merged**